src/spikeinterface/comparison/comparisontools.py

```diff
@@ -8,7 +8,7 @@
 import numpy as np
 
 
-def count_matching_events(times1, times2, delta=10):
+def count_matching_events(times1: list, times2: list, delta=10):
```

Comment:
Separate comment if you are adding typing you want to add one for delta as well. Is that only int or can it take float? If it is a float I would change default to 10.0 for us.